modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,355 @@
-= Section 2
+= Section 1
 
-This is _Section 2_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+== Introduction to Data Science Pipelines
```

Comment:
As per design doc, I think section 1 is  a general introduction to DS pipelines with basic terminology definitions and the types KF and ELyra and whatever else is coming into GA version. Section 2 should be about KF in particular and section 3 about ELyra.